examples/siro_sandbox/hydra_helper.py

```diff
@@ -13,7 +17,45 @@
 )
 
 
+class HabitatHitlConfigPlugin(SearchPathPlugin):
+    def manipulate_search_path(self, search_path: ConfigSearchPath) -> None:
+        search_path.append(
+            provider="siro_sandbox",
+            path="file://examples/siro_sandbox/config",
+        )
+
+
 def register_hydra_plugins():
     register_hydra_plugin(HabitatConfigPlugin)
     register_hydra_plugin(HabitatBaselinesConfigPlugin)
-    # todo: also register search paths for hitl framework's configs
+    register_hydra_plugin(HabitatHitlConfigPlugin)
+
+
+class ConfigObject:
+    def __init__(self, **entries):
+        self.__dict__.update(entries)
+
+
+def omegaconf_to_object(cfg):
```

Comment:
The motivation here is runtime perf. The idea is to call this once your config is loaded and finalized and you no longer need OmegaConf's fancy features. We should do something similar for habitat-lab and baselines in the future.

Reply:
This is great knowledge. Thanks for leading the way here.